satpy/writers/cf_writer.py

```diff
@@ -314,6 +314,8 @@ def default(self, obj):
             for key, val in obj.items():
                 serialized[key] = self.default(val)
             return serialized
+        elif isinstance(obj, np.ndarray) and not obj.shape:
+            return float(obj)
```

Comment:
Isn't there a `.item()` method on numpy arrays to get the scalar value?

Reply:
Thanks, modified.